src/ui/webeiddialog.cpp

```diff
@@ -60,18 +72,32 @@ WebEidDialog::WebEidDialog(QWidget* parent) : WebEidUI(parent), ui(new Private)
         QFile f(QStringLiteral(":dark.qss"));
         if (f.open(QFile::ReadOnly | QFile::Text)) {
             setStyleSheet(styleSheet() + QTextStream(&f).readAll());
-            ui->selectCertificateOriginLabelIcon->setPixmap(pixmap(QLatin1String("origin")));
-            ui->pinInputOriginLabelIcon->setPixmap(pixmap(QLatin1String("origin")));
-            ui->cardChipIcon->setPixmap(pixmap(QLatin1String("no-id-card")));
-            ui->fatalErrorIcon->setPixmap(pixmap(QLatin1String("fatal")));
-            ui->aboutIcon->setPixmap(pixmap(QLatin1String("fatal")));
-            ui->helpButton->setIcon(QIcon(QLatin1String(":/images/help_dark.svg")));
+            ui->selectCertificateOriginLabelIcon->setPixmap(pixmap("origin"_L1));
+            ui->pinInputOriginLabelIcon->setPixmap(pixmap("origin"_L1));
+            ui->cardChipIcon->setPixmap(pixmap("no-id-card"_L1));
+            ui->fatalErrorIcon->setPixmap(pixmap("fatal"_L1));
+            ui->aboutIcon->setPixmap(pixmap("fatal"_L1));
+            ui->helpButton->setIcon(QIcon(QStringLiteral(":/images/help_dark.svg")));
```

Comment:
```suggestion
            ui->helpButton->setIcon(QIcon(u":/images/help_dark.svg"_s));
```

Reply:
I did not feel safe create QString literal because using private api